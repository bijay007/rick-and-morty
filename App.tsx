import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

export default function App() {
  console.log('working here...')
  return (
    <View style={styles.container}>
      <Text>Welcome to the adventures of Rick and Morty</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
