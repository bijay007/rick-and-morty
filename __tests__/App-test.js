import React from 'react';
import * as renderer from 'react-test-renderer';
import App from '../App.tsx';

describe('App', () => {
  beforeAll(() => {
    jest.useFakeTimers();
  });

  it('renders the ActivityIndicator component before font loads', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree.type).toBe('ActivityIndicator');
    expect(tree.children).toBeNull();
  });
});
