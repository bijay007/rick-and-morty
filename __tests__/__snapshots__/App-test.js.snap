// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the ActivityIndicator component before font loads 1`] = `
<ActivityIndicator
  animating={true}
  color="#999999"
  hidesWhenStopped={true}
  size="small"
/>
`;
